### PR TITLE
Fix for issue #1324: Use lowerCamelCase for FirefoxAccountsError enum…

### PR DESCRIPTION
**Pull request by hardik25**

### CODE REVIEW

File: components/fxa-client/ios/FxAClient/Errors/FxAError.swift [hunk 1]
```diff
@@ -8,22 +8,22 @@ import Foundation
 // swiftlint:disable identifier_name
```
Comment (eoger) [inline components/fxa-client/ios/FxAClient/Errors/FxAError.swift]: You need to delete these two lines for the "Check Swift formatting" to succeed.

File: CHANGES_UNRELEASED.md [hunk 2]
```diff
@@ -3,3 +3,10 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.32.1...master)
+## FxAClient
```
Comment (eoger) [inline CHANGES_UNRELEASED.md]: Nit: add a newline before the title

Comment (hardik25) [reply CHANGES_UNRELEASED.md]: Added a new line. Thanks!

File: components/fxa-client/ios/FxAClient/Errors/FxAError.swift (hunk 1 above)
Comment (hardik25) [reply components/fxa-client/ios/FxAClient/Errors/FxAError.swift]: Done. Test passed. Thanks!

**Comment by hardik25** [comment]
@eoger, Could you please review the latest patch? I've made your suggested changes.

**Comment by eoger** [comment]
I'm seeing some of my commits in the commits list, I believe your rebase went wrong. Could you rebase again? Thanks!

**Pull request closed without merging**